.travis.yml

```diff
@@ -171,106 +163,3 @@ jobs:
         - TEST='raiden/tests/integration/cli'
         - TRANSPORT_OPTIONS='--transport=matrix'
         - RUN_SYNAPSE=1
-
-    - stage: deploy
```

Comment:
This means that we will not have build deployment on neither travis nor circle ci until the builld is implemented on circleCI right? if so, how soon can we get the builds up and running on circle CI? i imagine a release is going to be made during this week.

Reply:
Releases via CircleCI are already implemented, just skipped until the github token is added to the context. (MacOS binaries are still missing on CircleCI though.)